public/manifest.json

```diff
@@ -1,25 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
+    "short_name": "Backdoor",
+    "name": "Backdoor to Frigidsec",
+    "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
+        "src": "/android-chrome-192x192.png",
+        "sizes": "192x192",
+        "type": "image/png"
     },
     {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
+        "src": "/android-chrome-512x512.png",
+        "sizes": "512x512",
+        "type": "image/png"
     }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+    ],
+    "theme_color": "#E95678",
```

Comment:
Change theme color to #F43E5C

Reply:
I tried that color but I think it was too bright and the link embed color will not look good, so I picked lighter one.

Reply:
I tried that color but I think it was too bright and the link embed color will not look good, so I picked lighter one.

Reply:
Updated the color though